crates/test-collector/src/lib.rs

```diff
@@ -63,10 +64,30 @@ pub enum ExpectedTestResult {
 
 #[derive(Debug, Clone, PartialEq)]
 pub enum ForkConfig {
+    Id(String),
+    Params(Url, BlockId),
+}
+
+#[derive(Debug, Clone, PartialEq)]
+enum RawForkConfig {
     Id(String),
     Params(String, BlockId),
 }
```

Comment:
Do we really need to have another struct for handling parsing URL errors?

Reply:
I found it the easiest to do while retaining the current interface. Can refactor it if you want to